metadata-ingestion/setup.py

```diff
@@ -243,7 +243,8 @@ def get_long_description():
 }
 
 databricks = {
-    "databricks-sdk>=0.1.1",
+    # 0.1.11 appears to have authentication issues with azure databricks
+    "databricks-sdk>=0.1.1, <0.1.11",
```

Comment:
if it's going to be fixed in the next release, we can also change it to a `!=` restriction instead

Reply:
Not confident that'll happen, but will monitor so we can eventually drop the pin